src/components/Inputs/Select.tsx

```diff
@@ -82,14 +83,19 @@ function Select<T>({
     }
   }, [options, setInputValue, value])
 
+  const onSelectContainerClick = () => {
+    setShowPopup(true)
+    setFocused(true)
+  }
+
   return (
     <>
       <SelectContainer
         variants={sectionChildrenVariants}
         animate={canBeAnimated ? (!disabled ? 'shown' : 'disabled') : false}
         onAnimationComplete={() => setCanBeAnimated(true)}
         custom={disabled}
-        onClick={() => setShowPopup(true)}
+        onClick={onSelectContainerClick}
```

Comment:
Nitpicking (really): I like to prefix my "action" callbacks with "handle". `handleSelectContainerClick`.

Reply:
Hm, and I like to prefix actions with `on` which is more of the HTML standard 😅 Since it's a nitpick, can we just leave it as it is? 😇

Reply:
(FWIW I prefer `on` as well, it's sticking to event terminology.)

Reply:
Alright! Just a couple of resources on the subject:

- https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364
- https://reactjs.org/docs/handling-events.html
- https://jaketrent.com/post/naming-event-handlers-react

Reply:
event handler removed, cause it's not needed :P